Unix/wsman/wsmanerrorhandling.c

```diff
@@ -1532,10 +1541,22 @@ MI_Result GetWsmanErrorFromSoapFault(
     {
         // No match found using detail, so try again w/o
         return GetWsmanErrorFromSoapFault(soapFaultCode, soapFaultSubcode,
-                                          NULL, wsmanError);
+                                          NULL, wsmanError, wsmanErrorStr);
     } 
     else 
     {
         return MI_RESULT_FAILED;
     }
 }
+
+const MI_Instance* 
+GetWsmanCimError(ERROR_TYPES type, OMI_Error *error)
+{
+    if (ERROR_WSMAN_OPERATION_TIMEDOUT == type)
+    {
+        OMI_Error_Set_ProbableCause(error, 111);   // timedout

```

Comment:
replace magic number with a #define - something that will be more necessary when we have more